docs/spec/v1/gitrepositories.md

```diff
@@ -433,6 +433,55 @@ GitRepository, and changes to the resource or in the Git repository will not
 result in a new Artifact. When the field is set to `false` or removed, it will
 resume.
 
+### Proxy secret reference
+
+`.spec.proxySecretRef.name` is an optional field used to specify the name of a
+Secret that contains the proxy settings for the object. These settings are used
+for all remote Git operations related to the GitRepository.
+The Secret can contain three keys:
+
+- `address`: The address of the proxy server. This is a required key.
+- `username`: The username to use if the proxy server is protected by basic
+   authentication. This is an optional key.
+- `password`: The password to use if the proxy server is protected by basic
+   authentication. This is an optional key.
```

Comment:
what difference are you exactly pointing to? is it the period at the prefix? i didn't add them to the proxy docs since they are secret keys and not API fields